Description: Remove unused build dep and features
 These features are used upstream to contol build options for the embedded
 copy of libmimalloc, but this package is patched to use the system
 libmimalloc.
Last-Update: 2025-03-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,2 +25,0 @@
-[build-dependencies.cc]
-version = "1.0"
@@ -30,2 +30,0 @@
-debug = []
-debug_in_debug = []
@@ -40,3 +40,0 @@
-local_dynamic_tls = []
-no_thp = []
-override = []
-secure = []
